src/graphql/mutations/hooks/patients.tsx

```diff
@@ -0,0 +1,95 @@
+import { useMutation } from "@apollo/react-hooks";
+import { ADD_PATIENT, EDIT_PATIENT } from "../templates/patients";
+import { CCPType } from "../../queries/templates/collectionPoints";
+import { triageLevel, status } from "../../queries/templates/patients";
+
+export function addPatientMutation(
+  {
+    gender,
+    age,
+    runNumber,
+    barcodeValue,
+    collectionPointId,
+    status,
+    triageCategory,
+    triageLevel,
+    notes,
+    transportTime,
+  }: {
+    gender: string;
+    age: number;
+    runNumber: number;
+    barcodeValue: number;
+    collectionPointId: CCPType;
+    status: status;
+    triageCategory: number;
+    triageLevel: triageLevel;
+    notes: string;
+    transportTime: Date;
+  },
+  complete: boolean,
+  setComplete: (val: boolean) => any
+) {
+  const [addPatient, { data }] = useMutation(ADD_PATIENT);
+
+  if (complete) {
+    addPatient({
+      variables: {
+        gender,
+        age,
+        runNumber,
+        barcodeValue,
+        collectionPointId,
+        status,
+        triageCategory,
+        triageLevel,
+        notes,
+        transportTime,
+      },
+    });
+    setComplete(false);
+  }
+}
+
+export function editPatientMutation({
```

Comment:
Correct me if i'm wrong, but I think this `editPatientMutation` function is unused? Please remove it